ocm.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/python
+import requests
+
+def call_ocm(endpoint, token, parameters=None):
+    requests.packages.urllib3.disable_warnings()
+    r = requests.get("{}/{}".format("https://api.openshift.com/api/", endpoint),
+            params=parameters, 
+            headers = {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer {}'.format(token),
+                'User-Agent': 'pythonCall'
+                }
+            )
+
+    if r.status_code == 204:
+        return []
+    if r.status_code != 200:
+        raise Exception('''looking up infomation from: {}\n
+                Error Code: {}\n {}'''.format(endpoint, r.status_code, r.url))
```

Comment:
can we raise `ValueError(r)` here?  Or some more-specific error?  And then catch that and log it in `handle_cluster`?

Reply:
Sorry, not super familiar with what you are asking for here. Do you mind elaborating a bit @wking ?

Reply:
this code is down in a low-level, and it's doing some string formatting to set things up for eventually being logged further up (or just crashing the script ;).  I'd rather skip the string formatting and leave the data structured down here:

```python
if r.status_code != 200:
    raise ValueError({'response': r})
```

and then up in `handle_cluster`:

```python
try:
    account = ocm.cluster_to_account(token=ocm_token, cluster=cluster)
except ValueError as error:
    if len(error.args) == 1:
        details = error.args[0]
    else:
        logger.error('unrecognized cluster_to_account error: {}'.format(error))
        return

    response = details.get('response')
    if not response:
        logger.error('cluster_to_account error had no request: {}'.format(error))
    return client.chat_postMessage(
            channel=channel,
            text='Failed to find {}: {} -> {} {}'.format(cluster, response.url, response.status_code, response.body.read())
```
or some such.